### DISCUSSION TITLE
How do I clip content to insert it under a particular header?

### DISCUSSION THREAD

**Discussion by lesserror-hxr** [coddingtonbear/obsidian-web#51] [Q&A] [answered]
Hi, how can I append content between different pages? Select the content to be appended within different intervals of different local pages.

**Comment by coddingtonbear** [comment, marked as answer]
This plugin can do anything that the [obsidian-local-rest-api](https://github.com/coddingtonbear/obsidian-local-rest-api) plugin supports, including allowing you to insert content within a page between whatever headers in a note.  There isn't a pre-defined template for that kind of thing, though, because of course  inserting content in between headers on a single page requires that the template know at least one of your page's headers, and it's not likely anybody shares the same header structure for any of their notes.

What you'll want to do is look at the documentation for any of the items marked `PATCH` here: https://coddingtonbear.github.io/obsidian-local-rest-api/ and create a new template in obsidian-web using that method.

For a concrete example -- my daily note looks like this:

```
# Plan

# Events

# Insights

# Discoveries

# Pomodoros
```

and I have two templates set up in obsidian-web -- one for capturing a "Discovery" and one for capturing an "Insight".  Here's what the discovery one looks like:

![image](https://user-images.githubusercontent.com/527661/196992848-8728a8f1-5be4-4760-bf42-1e5a71407e27.png)

The above template will insert content into my current daily note under the "Discoveries" header.  The one for capturing an "Insight" is the same except that the HTTP Headers field reads instead `Heading: Insights`.

Hopefully this helps!  Good luck.
